util/metrics/counter.go

```diff
@@ -36,6 +36,18 @@ func MakeCounter(metric MetricName) *Counter {
 	return c
 }
 
+func NewCounter(name, desc string) *Counter {
```

Comment:
It would be cleaner if you were to pack the name&desc in a `MetricName` and pass it to `MakeCounter`